src/services/offerService.js

```diff
@@ -89,13 +94,16 @@ export const hideOffer = measureTime(TIMED_ACTIONS.OFFER_HIDE, async (offerId) =
 
     } catch (error) {
 
-        if (!isErrorRegistered) createEvent(EVENT_TYPES.ERROR(
+        const errorArray = Array.isArray(error) ? error :
+            [{ msg: "Unexpected Error. Please try again later." }];
+
+        if (!isErrorRegistered) createErrorEvent(
```

Comment:
when the function call spans over multiple lines, it's better to have the `{}` around the if body